var/spack/repos/builtin/packages/py-langsmith/package.py

```diff
@@ -13,6 +13,7 @@ class PyLangsmith(PythonPackage):
 
     license("MIT")
 
+    version("0.1.1", sha256="10ff2b977a41e3f6351d1a4239d9bd57af0547aa909e839d2791e16cc197a6f9", expand=False)
```

Comment:
I didn't include this in this PR, but I needed an update of this package for another project.  I can take this line back out if needed.